sdc/tests/test_series.py

```diff
@@ -1295,6 +1295,50 @@ def test_impl(A):
         pd.testing.assert_series_equal(
             hpat_func(S), test_impl(S))
 
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice_loc_start(self):
+        def test_impl(A, n):
+            return A.loc[n:]
+        hpat_func = self.jit(test_impl)
+
+        all_data = [[1, 3, 5, 13, 22], [1, 3, 3, 13, 22], [22, 13, 5, 3, 1], [100, 3, 1, -3, -3]]
+        key = [1, 3, 18]
+        for index in all_data:
+            for n in key:
+                with self.subTest(index=index, start=n):
+                    S = pd.Series([2, 4, 6, 6, 3], index)
+                    pd.testing.assert_series_equal(hpat_func(S, n), test_impl(S, n))
+
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice_loc_stop(self):
+        def test_impl(A, k, n):
+            return A.loc[k:n]
+        hpat_func = self.jit(test_impl)
+
+        all_data = [[1, 3, 5, 13, 22], [1, 3, 3, 13, 22], [22, 13, 5, 3, 1], [100, 3, 0, -3, -3]]
+        key = [1, 3, 18]
+        for index in all_data:
+            k = index[0]
+            for n in key:
+                with self.subTest(index=index, start=k, stop=n):
+                    S = pd.Series([2, 4, 6, 6, 3], index)
+                    pd.testing.assert_series_equal(hpat_func(S, k, n), test_impl(S, k, n))
+
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice_loc_start_stop(self):
```

Comment:
Looks like test `test_series_slice_loc_start_stop` completely covers test `test_series_slice_loc_stop`. So we can remove test `test_series_slice_loc_stop`. Please correct me if I am wrong.

Reply:
test_series_slice_loc_stop tests functional only for stop position, but we have limitation for functional, therefore syntax [ : n ] not support in its implementation. User must write start element, if he want to get slice with beginning in first element

Reply:
Ok, is there a logical difference between tests `test_series_slice_loc_start_stop` and `test_series_slice_loc_stop`? As I see both tests cover `start = 1, 22, 100`.